eng/intellisense.targets

```diff
@@ -0,0 +1,32 @@
+<Project>
+
+  <PropertyGroup>
+      <!-- Search for the documentation file in the intellisense package and otherwise pick up the generated one. -->
+    <IntellisensePackageFolderName Condition="'$(IsPackable)' == '' or '$(IsPackable)' == 'false'">net</IntellisensePackageFolderName>
+    <IntellisensePackageFolderName Condition="'$(IsPackable)' == 'true'">dotnet-plat-ext</IntellisensePackageFolderName>
+    <LibIntellisenseDocumentationFilePath Condition="'$(UseIntellisenseDocumentationFile)' == 'true'">$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', '$(MSBuildProjectName)', '$(LibrariesConfiguration)', '$(NetCoreAppCurrent)', '$(MSBuildProjectName).xml'))</LibIntellisenseDocumentationFilePath>
```

Comment:
I don't think you should be defining this property to point at the built file.  Instead just rely on the value set by `DocumentationProjectOutputGroup` in the item `DocumentationProjectOutputGroupOutput`.

Let `LibIntellisenseDocumentationFilePath` only point to the nuget-package sourced version of the xml file.

You don't need to have this property be conditioned on `UseIntellisenseDocumentationFile` - that's redundant.  The only thing that needs to check `UseIntellisenseDocumentationFile` is the target that swaps the built xml with the package one.

Reply:
Agreed. We don't want to hardcode the output paths anywhere as they are subject to change.